src/docs/JobPlanner/references/JobPlannerServiceReference.adoc

```diff
@@ -299,4 +299,145 @@ In such case, an *Association failed* notification will be triggered to users su
+
+To update an authorization, the admin needs to click on the drop-down list of the access type or the priority level and select a new value.
+The figure below shows an example.
+
+image::../../images/Update-association-authorisation.png[align=center]
+
+Once the selection is made the authorization will be updated as shown in the figure below.
+
+image::../../images/Updated-association-authorisation.png[align=center]
+
+To manage authorization for all associations owned by a specific user, the process is nearly identical.
+The admin or the associations' owner need to click on the dedicated button above the associations view in the Job-Planner calendar associations portal will be activated.
+The button has a shield-like with a user icon, as shown in the figure below.
+
+image::../../images/Association-owner-button.png[align=center]
+
+A dedicated authorization view will open.
+The figure below shows an example.
+
+image::../../images/Association-owner-view.png[align=center]
+
+Each authorization has the same attributes as an association authorization, however, only the association ID is replaced by `Owner`.
+This attribute is used to give access to all associations created by the specified username.
+
+To create a new associations' owner authorization, the admin just needs to click on the "+" button.
+Then the view of creating authorization (as shown in the figure below) will be popped up.
+The admin needs to fill in the authorization attributes (as presented above), then click the "Add authorization" button.
+The figure below shows an example of an authorization which gives the user named `Joseph` `write` access to all of `admin`'s authorizations.
+
+image::../../images/Create-association-owner-autthorisation.png[align=center]
+
+Once the admin has created an authorization, it will appear in the authorizations list in the authorization view, as shown in the figure below.
+
+image::../../images/List-of-owner-authorisations.png[align=center]
+
+The remaining actions such as edit and delete are identical to the association authorization actions.
+
+=== Authorization calculation rules
+
+Since each user might belong to multiple user groups, a user could have multiple authorizations over an association or a group of them due to his/her username or user group(s) authorizations.
+In such case, the resulting access type will be calculated as follows:
+
+* If the username-assigned authorization for the association exists, it is prioritized and its access type will be the user's resulting rights over the association.
+
+* If the previous case does not exist, the username-assigned associations' owner authorization access type will be the resulting user's rights over the association.
+
+* If multiple user-groups association authorizations exist for the selected association, without the previous cases, the resulting user's rights over the association will be the access type of the group authorization that have the highest priority.
```

Comment:
If multiple user-groups association authorizations exist for the selected association, and none of the previous cases apply, the resulting user's rights over the association will be the access type of the group authorization that has the highest priority.